### PR TITLE
prow: Add ipam-extensions basic config

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by oshoval** [comment]
/cc @dhiller @maiqueb @qinqon 

Thanks

File: github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml [hunk 1]
```diff
@@ -136,6 +136,13 @@ plugins:
     - owners-label
     - trigger
 
+  kubevirt/ipam-extensions:
+    plugins:
+    - approve
+    - lgtm
+    - owners-label
+    - trigger
```
Comment (qinqon) [inline github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml:144]: We don't need trigger, since we don't have prow lanes.

**Review by qinqon — CHANGES_REQUESTED** [review]

File: github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml (hunk 1 above)
Comment (oshoval) [reply github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml]: Removed

imho it doesnt hurt it was there (anyhow removed)

**Comment by oshoval** [comment]
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/3492/pull-kubevirt-org-github-config-updater/1805867032248324096

```
{"component":"peribolos","error":"failed to list ipam-extensions(ipam-extensions) members: return code not 2XX: 404 Not Found","level":"warning","msg":"failed to update ipam-extensions members: failed to list ipam-extensions(ipam-extensions) members: return code not 2XX: 404 Not Found","severity":"warning","time":"2024-06-26T08:43:27Z"}
```

maybe because OWNERS file is needed?

**Comment by dhiller** [comment]
> [prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/3492/pull-kubevirt-org-github-config-updater/1805867032248324096](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/3492/pull-kubevirt-org-github-config-updater/1805867032248324096)
> 
> ```
> {"component":"peribolos","error":"failed to list ipam-extensions(ipam-extensions) members: return code not 2XX: 404 Not Found","level":"warning","msg":"failed to update ipam-extensions members: failed to list ipam-extensions(ipam-extensions) members: return code not 2XX: 404 Not Found","severity":"warning","time":"2024-06-26T08:43:27Z"}
> ```
> 
> maybe because OWNERS file is needed? or because they need to be defined somewhere else

No, that's just because the team doesn't exist yet - a misconfiguration on the peribolos checks.

I'll take a look at the PR now.

**Comment by oshoval** [comment]
Thanks !

File: github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml [hunk 2]
```diff
@@ -500,6 +504,14 @@ orgs:
...
         repos:
           kubesecondarydns: admin
+      ipam-extensions:
+        description: ""
+        maintainers:
+          - maiqueb
+          - qinqon
+        privacy: secret
+        repos:
+          ipam-extensions: admin
```
Comment (dhiller) [inline github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml:514]: nit: as stated on the mailing list, we should cut down on repo access as far as possible.

**Review by dhiller — APPROVED** [review]
/override pull-kubevirt-org-github-config-updater

As said, the team doesn't exist yet, which is making the presubmit fail.

/approve

One nit inline.

File: github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml (hunk 2 above)
Comment (oshoval) [reply github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml:514]: what change is required then ? to keep just maiqueb ?
at least maiqueb should have write access please (for example for tags)

at least for now, we can iron the details bit later as agreed please

Note please that we would still need to be able to create tags / merge (unless tide takes care of that)
and to make sure github internal branch protection is enabled etc

**Comment by oshoval** [comment]
Thanks @qinqon @dhiller

**Pull request merged**